Routing/Loader/RestXmlCollectionLoader.php

```diff
@@ -150,6 +150,26 @@ protected function parseRoute(RouteCollection $collection, \DOMElement $node, $p
             $node->appendChild($defaultFormatNode);
         }
 
+        $nodes = clone $node;
+        for ($i = 0; $i < $nodes->childNodes->length; $i++) {
+            $loopNode = $node->childNodes->item($i);
+            if ($loopNode->nodeType == XML_TEXT_NODE) {
+                continue;
+            }
+
+            $newNode = $node->ownerDocument->createElementNS(
+                self::NAMESPACE_URI,
+                $loopNode->nodeName,
+                $loopNode->nodeValue
+            );
+
+            foreach ($loopNode->attributes as $value) {
+                $newNode->setAttribute($value->name, $value->value);
+            }
+
+            $node->appendChild($newNode);
+        }
```

Comment:
why did you need to add this code if this PR is just about syncing the xml schema?